x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -52,11 +52,11 @@ export class BulkUploader {
     });
 
     this.kibanaStats = kibanaStats;
-    this.kibanaStatusGetter = null;
-  }
 
-  setKibanaStatusGetter(getter) {
-    this.kibanaStatusGetter = getter;
+    this.kibanaStatus = null;
```

Comment:
nit: I don't think you need to do this, since the constructor is only executed once (and the default value is already `undefined`)